docs/adr/0001-actors.rst

```diff
@@ -0,0 +1,110 @@
+ADR 0001 - Actors
+=================
+
+Status
+------
+
+Proposed
+
+
+Context
+-------
+
+There are different kinds of actors that need to interact with Marsha.
+
+First we have the people managing a Marsha instance.
+
+Then we have people linking their own website (this website is a "consumer site") to a Marsha instance, to host videos.
+
+These consumer sites can host many customers. We call these "organizations". And these organizations have managers that can administrate some things about authors, video sharing between authors...
```

Comment:
technically it's more "publishers" than "customers". The customers of the consumer sites being the consumers.